pip/qsharp/_fs.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import os
+
+
```

Comment:
All functions missing return and param types

Reply:
Thanks for cleaning this up!